### ISSUE TITLE
How to train 16 frames TSM on something-something v2

### ISSUE THREAD

**Issue by zyc00** [mit-han-lab/temporal-shift-module#208]
I use this to train 16 frames TSM on something-something v2, but validation accuracy is only 61%, and test accuracy with twice sample is only 62.26%

python3 main.py somethingv2 RGB --arch resnet50 \
--num_segments 16 --gd 20 --lr 0.04 --lr_steps 20 30 --epochs 40 \
--batch-size 256 -j 16 --dropout 0.5 --consensus_type=avg --eval-freq=1 \
--shift --shift_div=8 --shift_place=blockres

How to get the same performance as the paper?
